### ISSUE TITLE
SSI Nitrate Detection Limit

### ISSUE THREAD

**Issue by SierraStreamsInstitute-Lab** [thosmos/riverdb#37]
The detection limit of our nitrate tests is 0.23. For results that fall under this detection limit can we enter "<0.23" or do we have to enter 0 or 0.23 (not necessarily true) or NA (confusing)?

**Comment by thosmos** [comment]
Currently the Field Measure tests are numeric only, so it's not possible to enter "<0.23" or "NA".  This is mainly for ensuring valid data.  The backend does have support for the same result qualifiers that are implemented on lab results, which enable adding an extra "<" to the value.  We could either add the result qualifiers to field measures or implement multiple test results to labs.  I would lean more towards the former because field measurements generally have a value.  But these issues also decrease the different between field measurements and lab measurements.  Maybe we just combine them and call them "Device Measurements" and keep the lab section more for tracking samples sent to external labs.

**Comment by SierraStreamsInstitute-Lab** [comment]
Yes, we could lump everything under device measurements and add qualifiers. 
We would still need to add log numbers for our nutrient and bacteria parameters though. 
We would also need to add new devices for the nutrient and bacteria measurements.

**Comment by thosmos** [comment]
Is the log number the same for all 3 measurements of the same parameter?  So would that go at the end of the line?

**Comment by SierraStreamsInstitute-Lab** [comment]
Yes, the log number is the same for all three measurements for nitrates and phosphates. At the end of the line would be great!
Bacteria just has one measurement.

**Comment by thosmos** [comment]
Would a general purpose notes field per line be good enough for the log #?
How is that data used?  Do you need it to have its own column in the data
table?

T

**Comment by SierraStreamsInstitute-Lab** [comment]
Hmm, I think we just want to be able to track the log number if there is a data quality issue or something. As long as it's associated with that line of measurements when we download data from RiverDB, that should be good enough.

**Comment by SierraStreamsInstitute-Lab** [comment]
Bacteria actually works really well in labs...we could leave it there.
